code/modules/mob/living/simple_animal/abnormality/teth/fragment.dm

```diff
@@ -64,6 +69,11 @@
 	fragment.song()
 	return TRUE
 
+/mob/living/simple_animal/hostile/abnormality/fragment/Destroy()
+	QDEL_NULL(legs)
+	QDEL_NULL(particle_notes)
+	QDEL_NULL(particle_song)
```

Comment:
Same with the previous comment, these naturally fading would probably look better.

Reply:
You can now rebase and call `fadeout()` on the particle_emitter's and they shall naturally fade away

Reply:
Same here